docs/source/conf.py

```diff
@@ -88,15 +71,15 @@
 # non-false value, then it is used:
 #today = ''
 # Else, today_fmt is used as the format for a strftime call.
-today_fmt = '%B %d, %Y'
+#today_fmt = '%B %d, %Y'
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['whatsnew/pr']
+exclude_patterns = []
 
 # The reST default role (used for this markup: `text`) to use for all
 # documents.
-default_role = 'literal'
+#default_role = None
```

Comment:
You probably want to leave this one as 'literal' - that's what lets us use single backticks for inline monospace text.